### ISSUE TITLE
Support with Next.js v10.0.0?

### ISSUE THREAD

**Issue by CaptainChemist** [auth0/nextjs-auth0#186]
### Description

I'm putting this ticket in because when I try to install the latest version of the @auth0/nextjs-auth0 package with next.js v10.0.0, I receive the following error when I do an npm install:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: fullstack@1.0.0
npm ERR! Found: next@10.0.0
npm ERR! node_modules/next
npm ERR!   next@"^10.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer next@"^9.0.0" from @auth0/nextjs-auth0@0.16.0
npm ERR! node_modules/@auth0/nextjs-auth0
npm ERR!   @auth0/nextjs-auth0@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
Here's the information about the error

```
{
  "code": "ERESOLVE",
  "current": {
    "name": "next",
    "version": "10.0.0",
    "whileInstalling": {
      "name": "fullstack",
      "version": "1.0.0",
      "path": "/Users/user/dev/fullstack-prisma/fullstack"
    },
    "location": "node_modules/next",
    "dependents": [
      {
        "type": "prod",
        "name": "next",
        "spec": "^10.0.0",
        "from": {
          "location": "/Users/user/dev/fullstack-prisma/fullstack"
        }
      }
    ]
  },
  "edge": {
    "type": "peer",
    "name": "next",
    "spec": "^9.0.0",
    "error": "INVALID",
    "from": {
      "name": "@auth0/nextjs-auth0",
      "version": "0.16.0",
      "whileInstalling": {
        "name": "fullstack",
        "version": "1.0.0",
        "path": "/Users/user/dev/fullstack-prisma/fullstack"
      },
      "location": "node_modules/@auth0/nextjs-auth0",
      "dependents": [
        {
          "type": "prod",
          "name": "@auth0/nextjs-auth0",
          "spec": "*",
          "from": {
            "location": "/Users/user/dev/fullstack-prisma/fullstack"
          }
        }
      ]
    }
  },
  "peerConflict": null,
  "strictPeerDeps": false,
  "force": false
}
```

### Environment

> Please provide the following:

Next 10.0.0
@auth0/nextjs-auth0 0.16.0
Node 15.0.1
NPM 7.0.3

**Comment by pieeee** [comment]
npm install @auth0/nextjs-auth0 --force will fix the error

**Comment by CaptainChemist** [comment]
Thanks @pieeee! Have you confirmed that this works with Nextjs 10?

**Comment by pieeee** [comment]
@CaptainChemist however i installed this package with `--force` command. I don't now why, every-time I logged in,  I always get a null value from `session?.user`. Do you have any idea? I followed the instructions from readme.md? Please letme know if you integrate package successfully with next.js

**Comment by helloitsm3** [comment]
> 
> 
> @CaptainChemist however i installed this package with `--force` command. I don't now why, every-time I logged in, I always get a null value from `session?.user`. Do you have any idea? I followed the instructions from readme.md? Please letme know if you integrate package successfully with next.js

Currently using NextJS 10.0.3. Was able to login successfully and retrive user's information

**Comment by pieeee** [comment]
@helloitsm3 did you follow the instruction they described in the readme?

**Comment by helloitsm3** [comment]
> 
> 
> @helloitsm3 did you follow the instruction they described in the readme?

My code is somewhat similar. What happens when you try to access /api/profile?

**Comment by pieeee** [comment]
@helloitsm3 It always returns `null` from `session.user` even though I was logged in.

**Comment by pieeee** [comment]
> > @helloitsm3 did you follow the instruction they described in the readme?
> 
> My code is somewhat similar. What happens when you try to access /api/profile?

@helloitsm3 It always returns `null` from `session.user` even though I was logged in.

**Comment by helloitsm3** [comment]
> 
> 
> > > @helloitsm3 did you follow the instruction they described in the readme?
> > 
> > 
> > My code is somewhat similar. What happens when you try to access /api/profile?
> 
> @helloitsm3 It always returns `null` from `session.user` even though I was logged in.

Can you share some code so that we can help you to debug? I'm able to retrieve user information after logging in

**Comment by pieeee** [comment]
> > > > @helloitsm3 did you follow the instruction they described in the readme?
> > > 
> > > 
> > > My code is somewhat similar. What happens when you try to access /api/profile?
> > 
> > 
> > @helloitsm3 It always returns `null` from `session.user` even though I was logged in.
> 
> Can you share some code so that we can help you to debug? I'm able to retrieve user information after logging in

let me try again. If I failed, I will share

**Comment by pieeee** [comment]
> > > > @helloitsm3 did you follow the instruction they described in the readme?
> > > 
> > > 
> > > My code is somewhat similar. What happens when you try to access /api/profile?
> > 
> > 
> > @helloitsm3 It always returns `null` from `session.user` even though I was logged in.
> 
> Can you share some code so that we can help you to debug? I'm able to retrieve user information after logging in

let me try again. If I failed, I will share

**Comment by adamjmcgrath** [comment]
Hi @pieeee - are you still seeing issues?

If so please share the code that you are having issues with.

You could also try the new Beta here https://github.com/auth0/nextjs-auth0/releases/tag/v1.0.0-beta.0
